ui/redux/actions/comments.js

```diff
@@ -54,6 +57,23 @@ export function doCommentList(
     const creatorChannelClaim = getChannelFromClaim(claim);
     const { claim_id: creatorClaimId, name: channelName } = creatorChannelClaim || {};
 
+    if (!myChannelClaims) {
+      console.error('Failed to fetch channel list.'); // eslint-disable-line
+      return;
+    }
+
+    const myChannelClaim = myChannelClaims.find((x) => x.claim_id === requesterChannelId);
+    if (!myChannelClaim) {
+      console.error('You do not own this channel.'); // eslint-disable-line
+      return;
+    }
+
+    const channelSignature = await channelSignName(myChannelClaim.claim_id, myChannelClaim.name);
+    if (!channelSignature) {
+      console.error('Failed to sign channel name.'); // eslint-disable-line
+      return;
+    }
```

Comment:
:small_red_triangle: @saltrafael, can you help to review and fix 063b1178 overall?  Something broke, and I think you are more aware of `doCommentList` corner cases.

For this particular chunk:
- starting a `COMMENT_LIST_STARTED` without ending it with a counterpart
- the client code (and users) will have no idea what's happening when any of these hit.

Help double-check all callers aren't affected?

Quick fix:
Maybe just dispatching `COMMENT_LIST_FAILED` for these cases would be sufficient if no time to handle gracefully.

Long term fix:
I think this bad pattern is copied from `doCommentListOwn`.  It only works there due to limited/controlled use, and it does suffer from the "spinner forever" situation on failure. Been trying to fix these by making sure the error path is not neglected.  Something to keep in mind moving forward.